src/templates/ru/2019/base_ebook.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base/2019/base_ebook.html" %}
+
+{% block title %}Web Almanac за {{ year }} год от HTTP Archive{% endblock %}
+
+{% block date_published %}2020-05-10T12:00:00.000Z{% endblock %}
+{% block date_modified %}2020-07-20T00:00:00.000Z{% endblock %}
```

Comment:
These have been removed in later version (we handle dates in a different way).

```suggestion
```

However you will need to sync latest `main` into your branch or this will cause an error when you remove these.